### DISCUSSION TITLE
How to Create Binary Package for ffmpegthumbnailer

### DISCUSSION THREAD

**Discussion by americanpossum** [termux/termux-packages#7347] [Show and tell]
Hey there, guys,

I have modified ffmpegthumbnailer to compile on termux. The source is at https://github.com/americanpossum/ffmpegthumbnailer while the binaries are in the releases folder. 

I would like to submit this package to the main termux archives so that others may easily install this to help in their own projects. However, I am a bit unsure of how to submit a binary package. Could someone point me to the instructions?

**Comment by ghost** [comment]
> bit unsure of how to submit a binary package.

You can create the `build.sh` script (using the original sources, not your fork) and patches. Binary packages aren't accepted. Everything must be compiled from source through our build system. Exceptions are very rare and only made for platform-independent software coming from trusted source.

https://github.com/termux/termux-packages/wiki/Creating-new-package
